src/hotspot/share/utilities/vmError.cpp

```diff
@@ -1712,40 +1712,43 @@ void VMError::report_and_die(int id, const char* message, const char* detail_fmt
     }
   }
 
-  // Part 1: print an abbreviated version (the '#' section) to stdout.
+  // Part 1: print to screen
```

Comment:
What is a "screen"? We print to files/file-descriptors

Reply:
Probably "screen" means the console on display. It is description before JDK-8220786 was fixed.
Is it more accurate to change it to "stdout/stderr"?

Reply:
Yes please